internal/extramanifest/policy.go

```diff
@@ -0,0 +1,125 @@
+package extramanifest
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"log"
+	"sort"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	policyv1 "open-cluster-management.io/config-policy-controller/api/v1"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//+kubebuilder:rbac:groups=policy.open-cluster-management.io,resources=policies,verbs=get;list
+
+const ztpDeployWaveAnnotation = "ran.openshift.io/ztp-deploy-wave"
+
+// GetPolicies gets the policies matching the labels from the namespace
+func (h *EMHandler) GetPolicies(ctx context.Context, labels map[string]string) ([]*policiesv1.Policy, error) {
+	listOpts := []client.ListOption{
+		client.MatchingLabels(labels),
+	}
+
+	policies := &policiesv1.PolicyList{}
+	if err := h.Client.List(ctx, policies, listOpts...); err != nil {
+		return nil, err
+	}
+
+	var policyWaveMap = make(map[*policiesv1.Policy]int)
+
+	for _, policy := range policies.Items {
+		// Ignore policies with remediationAction enforce
```

Comment:
It's better if comments explain **why** we do things, as opposed to **what** we're doing (unless it's not obvious). 

In this particular case it's hard to know why you chose to ignore those policies without such comment. The commit message is also completely empty - if someone were to come back to this code with the hopes of changing it, they will be in the dark

Reply:
Copied code from TALM :stuck_out_tongue:

Reply:
Comment added